.dockerignore

```diff
@@ -117,3 +117,10 @@ signatures
 demo
 packages/*/test
 packages/*/fixtures
+
+.git 
+
+packages/**/coverage
+
+# nyc test coverage
+packages/**/.nyc_output
```

Comment:
Can you also skip the tests and fixtures files. Also from from the published npm packages.

Reply:
We already skip them in Docker: https://github.com/platformatic/platformatic/blob/multi-stage-dockerfile/.dockerignore#L118

..but not in `.npmignore`, adding.

Reply:
(done).